src/MongoDB.Driver/Linq/Translators/PredicateTranslator.cs

```diff
@@ -988,7 +997,14 @@ private FilterDefinition<BsonDocument> TranslatePipelineContains(PipelineExpress
                     var ienumerableInterfaceType = constantExpression.Type.FindIEnumerable();
                     var itemType = ienumerableInterfaceType.GetTypeInfo().GetGenericArguments()[0];
                     var serializedValues = field.SerializeValues(itemType, (IEnumerable)constantExpression.Value);
-                    return __builder.In(field.FieldName, serializedValues);
+                    if (string.IsNullOrEmpty(field.FieldName))
+                    {
+                        return new BsonDocument("$in", serializedValues);
```

Comment:
I'm not sure, maybe here it's better to create a new `In` method to the builder. Or add a param to the method which already exists?

Reply:
After discussing this with @rstam, we decided to stay this as is